backend/data/processors/conditions_preprocessing.py

```diff
@@ -432,6 +442,134 @@ def map_word_to_program_type(processed, regex_word, type):
         flags=re.IGNORECASE,
     )  # hard to capture a generic case?
 
+#adds handbook notes and removes phrasing
+def remove_extraneous_comm_data(conditions, processed):
+    note = ""
+    # Academic Standing note
+    processed, number_of_subs = re.subn(
+        r"Prerequisite: Students must be in Good Academic Standing<br/><br/>", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Students must be in Good Academic Standing."
+
+    processed, number_of_subs = re.subn(
+        r"on Good Standing and", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Students must be in Good Academic Standing."
+
+    processed, number_of_subs = re.subn(
+        r"Good Standing", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
```

Comment:
There's quite a bit of excessive whitespace in this file, at the end of lines. This is a pretty inconsequential nitpick but, it is one of those things that our pylint picks up.

If you're on vscode, you don't need to manually clean this, just open up the command palette and search for `Trim Trailing Whitespace`. `cmd` `k` + `cmd x` is a shortcut for it too.

---------
The pylint output (starts line 36 of the most recent check as of writing this):
> backend/data/processors/conditions_preprocessing.py:371:0: C0303: Trailing whitespace (trailing-whitespace)
backend/data/processors/conditions_preprocessing.py:466:0: C0303: Trailing whitespace (trailing-whitespace)
backend/data/processors/conditions_preprocessing.py:481:0: C0303: Trailing whitespace (trailing-whitespace)
backend/data/processors/conditions_preprocessing.py:487:0: C0303: Trailing whitespace (trailing-whitespace)
backend/data/processors/conditions_preprocessing.py:490:5: C0303: Trailing whitespace (trailing-whitespace)
backend/data/processors/conditions_preprocessing.py:497:5: C0303: Trailing whitespace (trailing-whitespace)
backend/data/processors/conditions_preprocessing.py:503:5: C0303: Trailing whitespace (trailing-whitespace)